README.md

```diff
@@ -70,11 +70,11 @@ If developing the SDK, you also need:
 
 ## Getting Started
 
-Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started.html) for an example of authenticating and making an API call using the SDK.
+Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started) for an example of authenticating and making an API call using the SDK.
```

Comment:
Line 25 says support coming with v1.0.  It's here now.

Reply:
Modified the sentence so it makes more sense now